rebuild/src/addon.h

```diff
@@ -0,0 +1,177 @@
+#ifndef BLST_TS_ADDON_H__
+#define BLST_TS_ADDON_H__
+
+#include <openssl/rand.h>
+
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <string_view>
+
+#include "blst.hpp"
+#include "napi.h"
+
+// TODO: these should come out post PR review
+using std::cout;
+using std::endl;
```

Comment:
This is a bad practice as it leaves space for clashes and order-dependent compilation. See https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive

Reply:
clearly didn’t not follow my own directions in the comment above and they didn’t get removed…. will do that now!!!